src/bag3_digital/layout/stdcells/gates.py

```diff
@@ -709,9 +719,9 @@ def draw_layout(self) -> None:
         self.add_pin('enb', self.connect_to_tracks(pports.g1, tid))
         mlm = MinLenMode.MIDDLE if separate_out else MinLenMode.NONE
         tid = TrackID(hm_layer, nout_tidx, width=tr_w_h)
-        nout = self.connect_to_tracks(nports.d, tid, min_len_mode=mlm)
+        nout = self.connect_to_tracks(nports.d, tid)
```

Comment:
Why did `mlm` get removed from these lines?

Reply:
This was either a typo or the mlm was causing a track end collision

Reply:
I need more clarity if we're going to remove a default. If this only happens in certain cases (e.g. in your non `vertical_mid` case), then please add an if-case and a comment about it.